### DISCUSSION TITLE
Developer Access

### DISCUSSION THREAD

**Discussion by Spazzinq** [Zrips/CMI#4143] [Q&A]
**Question: **
Hey, I'm the developer of FlightControl, and I was wondering whether I could have access to CMI to fix an issue in FlightControl caused by the elevator addon.

**Comment by Zrips** [comment]
Is there some specific issue with elevator feature which causes conflict with your plugin?

**Comment by Spazzinq** [comment]
Yeah, it enables flight and I wanted to do some debugging on my dev server to fix the temporary flight timer not enabling.
